packages/nimble-components/src/table/components/cell/styles.ts

```diff
@@ -34,4 +46,8 @@ export const styles = css`
         height: ${controlSlimHeight};
         align-self: center;
     }
+
+    .action-menu.focused {
```

Comment:
Similar to the tabindex stuff, might be worth a comment saying this class is currently configured by the keyboard nav manager. Right now I could see it getting refactored out since the template isn't manipulating this.

Reply:
Added comment.